ci/ray_ci/windows/build_base.sh

```diff
@@ -3,6 +3,8 @@
 set -ex
 
 conda init 
+# newer requests version is needed for python 3.9+
+conda install -q -y python="${PYTHON}" requests=2.31.0
```

Comment:
the `requests` package should be installed via pip rather than via conda?

could you provide more context on why "requests" version is needed?

Reply:
conda itself is broken without upgrading requests https://buildkite.com/ray-project/premerge/builds/15486#018cad7e-92d5-4d58-81d1-3c100afbd230/12648-12671